### ISSUE TITLE
Bug: VPN_SERVICE_PROVIDER ignored

### ISSUE THREAD

**Issue by Steve-Brule** [qdm12/gluetun#905]
### Is this urgent?

Yes

### Host OS

Debian 11

### CPU arch

x86_64

### VPN service provider

Mullvad

### What are you using to run the container

docker-compose

### What is the version of Gluetun

v3.27.1 built on 2022-03-21T21:01:57.916Z (commit 5cc55c9)

### What's the problem 🤔

VPN_SERVICE_PROVIDER is ignored and the container attempts to connect to EXPRESSVPN instead of the selected provider.

### Share your logs

```log
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version v3.27.1 built on 2022-03-21T21:01:57.916Z (commit 5cc55c9)

🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2022/03/22 03:27:03 INFO storage: merging by most recent 11100 hardcoded servers and 11096 servers read from /gluetun/servers.json
2022/03/22 03:27:03 INFO storage: Using ExpressVPN servers from file which are 3238h36m31s more recent
2022/03/22 03:27:03 ERROR failed validating VPN settings: OpenVPN settings validation failed: password is empty
2022/03/22 03:27:03 INFO Shutdown successful
```

### Share your configuration

```yml
version: "3"
services:
  gluetun:
    container_name: gluetun
    image: qmcgaw/gluetun
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
    environment:
      - BLOCK_ADS=off
      - BLOCK_MALICIOUS=off
      - BLOCK_SURVEILLANCE=off
      - DNS_KEEP_NAMESERVER=on
      - DNS_UPDATE_PERIOD=0
      - DOT=off
      - DOT_CACHING=off
      - DOT_IPV6=off
      - DOT_PROVIDERS=quad9
      - DOT_VALIDATION_LOGLEVEL=0
      - DOT_VERBOSITY=1
      - DOT_VERBOSITY_DETAILS=0
      - FIREWALL=on
      - FIREWALL_DEBUG=off
      - FIREWALL_INPUT_PORTS=5001,8888,9091
      - FIREWALL_VPN_INPUT_PORTS=XXXXX,XXXXX
      - HEALTH_TARGET_ADDRESS=debian.org:80
      - HEALTH_VPN_DURATION_ADDITION=5s
      - HEALTH_VPN_DURATION_INITIAL=10s
      - HTTPPROXY=on
      - HTTPPROXY_LISTENING_ADDRESS=:8888
      - HTTPPROXY_LOG=off
      - HTTPPROXY_STEALTH=on
      - HTTP_CONTROL_SERVER_ADDRESS=:8000
      - HTTP_CONTROL_SERVER_LOG=on
      - ISP=31173
      - OPENVPN_IPV6=on
      - OPENVPN_MSSFIX=0
      - OPENVPN_PROCESS_USER=nonrootuser
      - OPENVPN_PROTOCOL=udp
      - OPENVPN_USER=XXXXX
      - OPENVPN_VERBOSITY=1
      - OPENVPN_VERSION=2.5
      - OWNED_ONLY=yes
      - PGID=987
      - PUBLICIP_PERIOD=12h
      - PUID=994
      - SERVER_CITIES=gothenburg
      - SERVER_COUNTRIES=sweden
      - SERVER_HOSTNAMES=se-got-001,se-got-002,se-got-003,se-got-004,se-got-005,se-got-006,se-got-007,se-got-008,se-got-009,se-got-010,se-got-011,se-got-012
      - SHADOWSOCKS=on
      - SHADOWSOCKS_ADDRESS=:8388
      - SHADOWSOCKS_CIPHER=chacha20-ietf-poly1305
      - SHADOWSOCKS_LOG=off
      - SHADOWSOCKS_PASSWORD=XXXXX
      - TZ=Europe/Moscow
      - UPDATER_PERIOD=24h
      - UPDATER_VPN_SERVICE_PROVIDERS=mullvad
      - VERSION_INFORMATION=on
      - VPN_ENDPOINT_PORT=53
      - VPN_INTERFACE=tun0
      - VPN_SERVICE_PROVIDER=mullvad
    networks:
      - gluetun
    ports:
      - 127.0.0.1:8000:8000/tcp
      - 127.0.0.1:8888:8888/tcp
      - 8388:8388/tcp
      - 8388:8388/udp
    sysctls:
      - net.ipv6.conf.all.disable_ipv6=0
    volumes:
      - /media/mainstor/dockerdata/gluetun:/gluetun
networks:
  gluetun:
    name: gluetun
```

**Comment by leo15dev** [comment]
https://github.com/qdm12/gluetun/issues/904#issue-1176096907

v 3.28.1 worked well.

**Comment by Steve-Brule** [comment]
:latest doesn't work as expected with this container. Manually specifying v3.28.1 fixed it for me. Thanks.

**Comment by qdm12** [comment]
See https://github.com/qdm12/gluetun/issues/904#issuecomment-1074896256

> TLDR: You can re-use the latest image `qmcgaw/gluetun` safely, or `qmcgaw/gluetun:v3` to stay on `v3.28.1`.
